pkg/controller/ipam/antrea_ipam_controller_test.go

```diff
@@ -238,7 +249,28 @@ func TestReleaseStaleAddresses(t *testing.T) {
 	crdInformerFactory.Start(stopCh)
 
 	go controller.Run(stopCh)
+	controller.cleanupStaleAddresses()
+
+	// verify two stale entries were deleted, one updated to Reserved status
+	err := wait.PollImmediate(100*time.Millisecond, 2*time.Second, func() (bool, error) {
+		pool, err := poolLister.Get(pool.Name)
+		if err != nil {
+			return false, nil
+		}
 
-	// after cleanup pool should have single entry
-	verifyPoolAllocatedSize(t, pool.Name, poolLister, 1)
+		if len(pool.Status.IPAddresses) != 2 {
+			klog.InfoS("IP Pool status", "addresses", pool.Status.IPAddresses)
```

Comment:
test should use `t.Logf`, otherwise test log format would be inconsistent.

Reply:
Done, thanks